scripts/generate-schema.js

```diff
@@ -21,10 +21,8 @@ function generateSchema(options) {
     definitions: {
       optionsDefinition: {
         type: "object",
-        properties: options.reduce(
-          (props, option) =>
-            Object.assign(props, { [option.name]: optionToSchema(option) }),
-          {}
+        properties: Object.assign(
+          ...options.map(option => ({ [option.name]: optionToSchema(option) }))
         )
```

Comment:
I think no tests cover this part, need review.

Reply:
`tests_integration\__tests__\schema.js` seems to cover it.